.github/workflows/continuous-integration.yml

```diff
@@ -82,6 +82,18 @@ jobs:
         with:
           args: "xmllint --noout --schema vendor/squizlabs/php_codesniffer/phpcs.xsd lib/Doctrine/ruleset.xml"
 
+  lint-dockerfile:
+    name: "Lint Dockerfile files with Hadolint"
+    runs-on: "ubuntu-22.04"
+
+    steps:
+      - uses: "actions/checkout@v3"
+
+      - name: "Lint Dockerfile"
+        uses: "hadolint/hadolint-action@v3.1.0"
+        with:
+          dockerfile: ".docker/validate-against-schema/Dockerfile"
```

Comment:
This is going to run on every modification of every file. Maybe there are files that should not trigger this job?

Reply:
In this case, we should use a different workflow, as the `path` config is not allowed in individual jobs.
Please, confirm.